Doc/library/io.rst

```diff
@@ -318,7 +318,7 @@ I/O Base Classes
    This class provides empty abstract implementations for many methods
    that derived classes can override selectively; the default
    implementations represent a file that cannot be read, written or
-   seeked.
+   sought.
```

Comment:
Since this seems to be referring to the specific concept of "seek"ing in files rather than using it as just a word, I'm not sure if this change is correct, at least for me it feels like it would require more effort to interpret the meaning with this change.

I don't have anything to back this up though so others may disagree.

Reply:
I think it's okay, we also have "written" and not "writed".

Reply:
I'm having second thoughts...

@CAM-Gerlach Is "seeked" okay here?

Reply:
I much prefer "seeked" here

Reply:
Yeah, in this context "seeked" makes much more sense, IMO